inithelper_test.go

```diff
@@ -46,4 +51,48 @@ func TestInitHelper(t *testing.T) {
 		assert.Assert(t, logger == nil, "Logger is nil.")
 		assert.Assert(t, err != nil, "An error is expected. Found nil instead.")
 	})
+
+	t.Run("customWriter integration", func(t *testing.T) {
+		now := time.Now()
+		var buffer bytes.Buffer
+
+		logger, _ := InitHelper(InitOptions{})
+		logger.Out = &buffer
+		logger.WithTime(now)
+		logger.WithField("foo", "bar").Info("ciao")
+
+		type log struct {
+			Level   int    `json:"level"`
+			Message string `json:"msg"`
+			Foo     string `json:"foo"`
+			Time    int64  `json:"time"`
+		}
+
+		var result log
+		err := json.Unmarshal(buffer.Bytes(), &result)
+
+		assert.Assert(t, err == nil, "Unaxpected Error %s", err)
+		assert.DeepEqual(t, result, log{
+			Level:   int(30),
```

Comment:
```suggestion
			Level:   30,
```

`int` conversion is redundant